README.md

```diff
@@ -43,7 +43,20 @@ pip install jupyterhub-ltiauthenticator
 
     _Note_: Anyone with these two strings will be able to access your hub, so keep them secure!!
 
-4.  Pick a name for edX to call your JupyterHub server. Then, along with the two random strings you generated in step 3, paste them together to create an _LTI Passport String_ in the following format:
+4. By default, the user's name will be the `custom_canvas_user_id` passed in by canvas. If you
+   would like to use some other bit of information from the LTI request, you can pick what should
+   be used as the user id.
+
+   ```python
+   # Set the user's email as their user id
+   c.LTIAuthenticator.user_id_key = 'lis_person_contact_email_primary'
+   ```
+
+   A [partial list of keys in an LTI request](https://www.edu-apps.org/code.html#params)
+   is available to help. Your LMS provider might also implement custom keys
+   you can use.
+
```

Comment:
Wieeee I really appreciate the level on which you communicate here. It is easy to make a too generous assumption about the users knowledge and not introducing the user to when and why something makes sense to configure, but I think this section does an excellent job.

:heart: